components/PageTitle/PageTitle.module.scss

```diff
@@ -1,18 +1,14 @@
 .background {
     @include full-width;
```

Comment:
Since we don't have a background anymore, there is not point in having this container at all. So you can safely remove the class and the `div` in the component.

Reply:
updated